neptune/new/run.py

```diff
@@ -157,7 +161,28 @@ def assign(self, value, wait: bool = False) -> None:
         .. _assign docs page:
             https://docs.neptune.ai/api-reference/run#assign
         """
-        self[""].assign(value, wait)
+        self.root.assign(value, wait)
+
+    def fetch(self) -> dict:
+        """Fetch values of multiple fields as a dictionary.
+        The result will contain only Namspaces and attributes of Atom type.
```

Comment:
"The result will contain only Namspaces and attributes of Atom type." -> "The result will preserve the hierarchical structure of the runs metadata, but will contain only non-File Atom fields."
Externally we use word fields and fields only :) Attribute as a word is used only internally facing.